main/src/cgeo/geocaching/utils/AndroidRx2Utils.java

```diff
@@ -0,0 +1,99 @@
+package cgeo.geocaching.utils;
+
+import android.app.Activity;
+import android.os.HandlerThread;
+import android.os.Looper;
+import android.os.Process;
+
+import java.lang.ref.WeakReference;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Executors;
+
+import io.reactivex.Maybe;
+import io.reactivex.Observable;
+import io.reactivex.Scheduler;
+import io.reactivex.Single;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.functions.Consumer;
+import io.reactivex.functions.Predicate;
+import io.reactivex.internal.schedulers.RxThreadFactory;
+import io.reactivex.schedulers.Schedulers;
+
+public class AndroidRx2Utils {
+
+    public static final Scheduler computationScheduler = Schedulers.computation();
+
+    public static final Scheduler networkScheduler = Schedulers.from(Executors.newFixedThreadPool(10, new RxThreadFactory("network-")));
```

Comment:
The leak is not here, it happens to be on those threads. Anyway, `bindActivity` in its current form (from the old RxAndroid package)  is going away, and the new one being used when necessary (in `AndroidRx2Utils` as I am refactoring) uses a `WeakReference`.